src/components/ShareButton/ShareButton.e2e.ts

```diff
@@ -0,0 +1,30 @@
+import { browser } from 'protractor';
+import {
+  createStoryUrl,
+  waitForVisibilityOf,
+} from 'wix-ui-test-utils/protractor';
+import { shareButtonTestkitFactory } from '../../testkit/protractor';
+
+/**
+ * For tests containing interactions.
+ * Can be removed if not used.
+ * */
+describe('shareButton', () => {
+  const storyUrl = createStoryUrl({
+    kind: 'Components/Share',
+    story: 'ShareButton',
+    withExamples: true,
+  });
+  const dataHook = 'storybook-ShareButton';
+
+  beforeEach(() => browser.get(storyUrl));
+
+  it('should render', async () => {
+    const driver = shareButtonTestkitFactory({ dataHook });
+    await waitForVisibilityOf(
+      await driver.element(),
+      'Cannot find ShareButton',
+    );
+    expect((await driver.element()).isDisplayed()).toBe(true);
+  });
+});
```

Comment:
If all we have to test here is that it's rendering, I think we can remove this file.
However, we do need to add some kind of test for this button.
I'm assuming we'll need to mock the `navigator.share` api to test this, and we can probably do this in the enzyme test file.

Reply:
I delete e2e test, will mock navigator in spe